lib-injection/sources/sitecustomize.py

```diff
@@ -205,9 +206,12 @@ def _inject():
     integration_incomp = False
     runtime_incomp = False
     os.environ["_DD_INJECT_WAS_ATTEMPTED"] = "true"
+    spec = None
     try:
-        import ddtrace
-    except ImportError:
+        spec = importlib.util.find_spec("ddtrace")
+        if not spec:
```

Comment:
getting `None` back is a valid response.

I decided to just raise in that case to keep majority of this code the same as it was previously.